conrad-blucher-institute/Flare
# 5 create csv generator

Testing steps:
1. Build and raise containers
2.  Exec `docker exec flare-backend python3 ./main.py -c ./cspec/test_1-0-0.json`
3.  (op) Exec `docker exec flare-backend python3 ./main.py -c ./cspec/test_1-0-0.json -v` <-- verbose logging
4. You should see the csv appear both is your vscode and in the docker (because of the mount)
